examples/testing-dom__clipboard/cypress/integration/spec.js

```diff
@@ -0,0 +1,99 @@
+/// <reference types="cypress" />
+
+// access to the clipboard reliably works in Electron browser
+// in other browsers, there are popups asking for permission
+// thus we should only run these tests in Electron
+describe('Clipboard', { browser: 'electron' }, () => {
+  it('copies text to clipboard', () => {
+    cy.visit('index.html')
+    cy.get('code').trigger('mouseover')
+    cy.get('[aria-label="Copy"]').click()
+
+    // let's check the copied text
+    // on Chrome this operation will prompt the browser
+    // to ask the user for permission:
+    //  http://localhost:port wants to
+    //  See text and images copied the clipboard
+    cy.window().its('navigator.clipboard')
+    .invoke('readText')
+    .should('equal', 'npm install -D cypress')
+  })
+
+  it('shows the popup', () => {
+    cy.visit('index.html')
+    cy.get('code').trigger('mouseover')
+    cy.get('[aria-label="Copy"]').click()
+
+    cy.contains('.tinyToast', 'Copied!').should('be.visible')
+    // the toast then goes away in less than 2 seconds
+    cy.get('.tinyToast', { timeout: 2000 }).should('not.exist')
+  })
+
+  it('can set the clipboard text in the text area', () => {
+    cy.visit('index.html')
+    cy.get('code').trigger('mouseover')
+    cy.get('[aria-label="Copy"]').click()
+
+    // let's check the copied text
+    cy.window().its('navigator.clipboard')
+    .invoke('readText')
+    .should('equal', 'npm install -D cypress')
+    .then((text) => {
+      // paste the text from the clipboard into the text area
+      cy.get('#paste-here').click().invoke('val', text)
+    })
+
+    // take a screenshot to show the pasted text
+    cy.screenshot()
```

Comment:
I would remove these screenshots. It makes the test visually jump around for each screenshot. I was confused. It also makes it seem that screenshotting is necessary to see the pasted text, but I can see it in the UI.

Reply:
Ok, I will remove it, wanted to show the screenshots on CI, but not necessary if confusing